src/svelte/DialogBox.svelte

```diff
@@ -0,0 +1,37 @@
+<script context="module" lang="ts">
+  let instanceCounter = 0
+</script>
+
+<script lang="ts">
+  export let dialogBoxClosed: boolean = false
```

Comment:
```suggestion
  export let isOpen: boolean = true
```
The closed = false got me abit confused. Felt its easier to read using the true value.
I prefer to prefix booleans with `is`. Dont think you need `dialogBox` its already in the component name.

Reply:
Good point 👍